### PR TITLE
Fix PiecewisePolytropicFluid Python test, add some bindings

### LINKED ISSUE #4523: A test has label "unit.hydro.equationsofstate.python.piecewisepolytropicfluid"
From the ctest summary:
```
time                                                           =  96.77 sec*proc (111 tests)
unit                                                           = 1381.73 sec*proc (1355 tests)
unit.hydro.equationsofstate.python.piecewisepolytropicfluid    =   0.09 sec*proc (1 test)
utilities                                                      =  58.72 sec*proc (63 tests)
variablefixing                                                 =   4.28 sec*proc (5 tests)
visualization                                                  =   4.21 sec*proc (4 tests)
```

### PR DESCRIPTION
**Pull request by nilsvu**
## Proposed changes

Fixes #4523. I also added some bindings while I was touching the files.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: tests/Unit/PointwiseFunctions/Hydro/EquationsOfState/Python/Test_PolytropicFluid.py [hunk 1]
```diff
@@ -1,19 +1,27 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
-import spectre.PointwiseFunctions.Hydro.EquationsOfState as spectre_eos
-
 import unittest
```
Comment (pajkosmi) [inline tests/Unit/PointwiseFunctions/Hydro/EquationsOfState/Python/Test_PolytropicFluid.py:4]: @nilsvu are the imports required to be alphabetized in `.py` files, or is that just with `#include` in `.cpp`/`.hpp` files?  If not, then everything looks good to me.

Comment (nilsvu) [reply tests/Unit/PointwiseFunctions/Hydro/EquationsOfState/Python/Test_PolytropicFluid.py:4]: We haven't established a rule for sorting imports yet. I started to use isort recently, which puts system modules before third-party modules, then alphabetizes.

**Review by pajkosmi — DISMISSED** [review]
The ctest summary does not produce the long piecewise polytrope name anymore and isort has certain ordering for `.py` imports, so I approve.

File: src/PointwiseFunctions/Hydro/EquationsOfState/Python/Bindings.cpp [hunk 2]
```diff
@@ -7,7 +7,14 @@
 #include "PointwiseFunctions/Hydro/EquationsOfState/Python/PiecewisePolytropicFluid.hpp"
 #include "PointwiseFunctions/Hydro/EquationsOfState/Python/PolytropicFluid.hpp"
```
Comment (knelli2) [inline src/PointwiseFunctions/Hydro/EquationsOfState/Python/Bindings.cpp:8]: [Commit message]
This isn't a `PiecewisePolynomial` it's a `PiecewisePolytrope`

**Review by knelli2 — CHANGES_REQUESTED** [review]

**Review by nilsdeppe — COMMENTED** [review]
Looks good, just @knelli2's comment about the commit message. Your first commit still says "PiecewisePolynomial"

File: src/PointwiseFunctions/Hydro/EquationsOfState/Python/Bindings.cpp (hunk 2 above)
Comment (nilsvu) [reply src/PointwiseFunctions/Hydro/EquationsOfState/Python/Bindings.cpp:8]: 🤦‍♂️ fixed

**Review by knelli2 — APPROVED** [review]

**Review by nilsdeppe — APPROVED** [review]

**Pull request merged**